letsencrypt/client/standalone_authenticator.py

```diff
@@ -297,11 +290,9 @@ def already_listening(self, port):  # pylint: disable=no-self-use
                     "that port. Please stop the {0} program temporarily "
                     "and then try again.".format(name, pid, port))
                 return True
-        except (OSError, ValueError, IndexError):
-            # A sign that this command isn't available or usable this
-            # way on this operating system, or there was something
-            # unexpected about the format of the netstat output; we will
-            # not be able to recover from this condition.
+        except psutil.NoSuchProcess:
```

Comment:
Shall we also catch `psutil.AccessDenied`? http://pythonhosted.org/psutil/#psutil.net_connections